rust/hyperlane-core/src/types/message.rs

```diff
@@ -34,6 +36,34 @@ pub struct HyperlaneMessage {
     pub body: Vec<u8>,
 }
 
+impl Debug for HyperlaneMessage {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "HyperlaneMessage {{ id: {:?}, version: {}, nonce: {}, origin: {}, sender: {}, destination: {}, recipient: {}, body: 0x{} }}",
```

Comment:
Ah here we are, nice.

Thoughts on making origin/destination include the human readable domain name, if known?

Reply:
If you did this and I already missed it, ignore me

Reply:
Did it now